examples/cpp-dummy/micro_cpp_dummy.cpp

```diff
@@ -69,6 +69,21 @@ void MicroSimulation::reload_checkpoint()
     _micro_scalar_data = _checkpoint;
 }
 
+// This function needs to set the complete state of a micro simulation
+void MicroSimulation::setState(double micro_scalar_data, double checkpoint)
```

Comment:
Same as above, is micro_vector_data not needed to fully define the state?

Reply:
This comment was supposed to be at [line 83](https://github.com/precice/micro-manager/blob/841c32510cee3357a2122d7ae7460ec2638b001f/examples/cpp-dummy/micro_cpp_dummy.cpp#L83)